src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -15,6 +15,9 @@
 import seedu.address.model.person.ReadOnlyPerson;
 import seedu.address.model.person.exceptions.DuplicatePersonException;
 import seedu.address.model.person.exceptions.PersonNotFoundException;
+import java.util.Set;
```

Comment:
This import should be placed directly above:
`import java.util.function.Predicate;`